ui/app/models/mfa-login-enforcement.js

```diff
@@ -2,7 +2,30 @@ import Model, { attr, hasMany } from '@ember-data/model';
 import ArrayProxy from '@ember/array/proxy';
 import PromiseProxyMixin from '@ember/object/promise-proxy-mixin';
 import { methods } from 'vault/helpers/mountable-auth-methods';
+import { withModelValidations } from 'vault/decorators/model-validations';
+import { isPresent } from '@ember/utils';
 
+const validations = {
+  name: [{ type: 'presence', message: 'Name is required' }],
+  mfa_methods: [{ type: 'presence', message: 'At least 1 MFA method is required' }],
```

Comment:
NIT: as a design rule (one Ivana and I have chatted about) any number less <= to 10 should be spelled out. So here 1 should be spelled out to one.

Reply:
Sounds good thanks for letting me know. I'll go ahead and update.